libqpdf/QPDF_json.cc

```diff
@@ -420,12 +414,6 @@ QPDF::JSONReactor::containerEnd(JSON const& value)
         // Handle dangling indirect object references which the PDF spec says to treat as nulls.
```

Comment:
This comment is slightly confusing now since there's no code left.  Perhaps just changing `Handle` to `Ignore` would be adequate.

Reply:
I have taken the whole comment out. I think the comment only makes sense if you know what the code was previously, and does not add anything

Reply:
sounds good